interp/compiler.go

```diff
@@ -49,7 +49,10 @@ func (inst *instruction) String() string {
 		operands[i] = op.String()
 	}
 
-	name := instructionNameMap[inst.opcode]
+	name := ""
+	if int(inst.opcode) < len(instructionNameMap) {
+		name = instructionNameMap[inst.opcode]
+	}
```

Comment:
If this fails, that's most certainly a bug and it's rather unlikely this actually happens (it's rare that new instructions are added to LLVM, and it's unlikely they're used by TinyGo right away).

...but of course, adding an extra check won't harm here.